pynonymizer/database/mysql/query_factory.py

```diff
@@ -29,7 +29,10 @@ def _get_column_subquery(seed_table_name, column_strategy):
     elif column_strategy.strategy_type == UpdateColumnStrategyTypes.UNIQUE_LOGIN:
         return f"( SELECT {_RAND_MD5} )"
     elif column_strategy.strategy_type == UpdateColumnStrategyTypes.FAKE_UPDATE:
-        return f"( SELECT `{column_strategy.qualifier}` FROM `{seed_table_name}` ORDER BY RAND() LIMIT 1)"
+        column = column_strategy.qualifier
+        if column_strategy.sql_type:
+            column += "::" + column_strategy.sql_type
```

Comment:
i'm pretty sure this syntax won't work for mysql. It needs to be some variation on `CAST(...` or `CONVERT`.

Reply:
Sorry, you are right. I thought it was the same 😅 I will fix it ASAP